### PR TITLE
small spelling & grammar fixes

### PR DESCRIPTION
**Pull request by z89**
Fixed some small spelling & grammar mistakes. I also changed some nouns for consistency. Feel free to reject this PR if you prefer the original.

### CODE REVIEW

File: cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The impact of a successful CSRF attack is limited to the capabilities exposed by
 In short, the following principles should be followed to defend against CSRF:
 
 - **Check if your framework has [built-in CSRF protection](#use-built-in-or-existing-csrf-implementations-for-csrf-protection) and use it**
-    - **If framework does not have built-in CSRF protection add [CSRF tokens](#token-based-mitigation) to all state changing requests (requests that cause actions on the site) and validate them on backend**
+    - **If framework does not have built-in CSRF protection add [CSRF tokens](#token-based-mitigation) to all state changing requests (requests that cause actions on the site) and validate them on the backend**
```
Comment (szh) [inline cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md:12]:
```suggestion
    - **If framework does not have built-in CSRF protection, add [CSRF tokens](#token-based-mitigation) to all state changing requests (requests that cause actions on the site) and validate them on the backend**
```

Maybe you can sneak in a comma while you're at it :)

**Review by szh — DISMISSED** [review]
LGTM, just one tiny suggestion

File: cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md (hunk 1 above)
Comment (z89) [reply cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md:12]: I missed that one, thanks for the iteration :)

**Review by mackowski — APPROVED** [review]

**Comment by mackowski** [comment]
thanks @z89!

**Pull request merged**